kyo-core/shared/src/main/scala/kyo/Meter.scala

```diff
@@ -1,25 +1,13 @@
 package kyo
 
+import org.jctools.queues.MpmcUnboundedXaddArrayQueue
+import scala.annotation.tailrec
+
 /** A Meter is an abstract class that represents a mechanism for controlling concurrency and rate limiting.
   */
 abstract class Meter:
     self =>
 
-    /** Returns the number of available permits.
-      *
-      * @return
-      *   The number of available permits as an Int effect.
-      */
-    def available(using Frame): Int < IO
-
-    /** Checks if there are any available permits.
-      *
-      * @return
-      *   A Boolean effect indicating whether permits are available.
-      */
-    def isAvailable(using Frame): Boolean < IO =
```

Comment:
I've removed `isAvailable` since it doesn't seem very useful, renamed `available` to `availablePermits`, and moved the method to the end of the class.